contributor_docs/p5.strands.md

```diff
@@ -0,0 +1,251 @@
+<!-- How p5.strands JS-to-GLSL compilation works. -->
+
+# p5.strands Overview
+
+Shader programming is an area of creative coding that can feel like a dark art to many. People share lots of stunning visuals that are created with shaders, but shaders feel like a completely different way of coding, requiring you to learn a new language, pipeline, and paradigm.
```

Comment:
I think it would be really good to more directly express the intended audience. There's parts of this that would be useful to anyone encountering strands, and parts only useful to people who want to contribute to strands (though this is the contribution section on the site, the intro reads like a general motivation for strands usage). Ie, it's important all users of p5.strands are aware this is a "different mode" of javascript; and that there is also GLSL and OpenGL. However, I don't think terminology of transpiler is generally necessary unless people are interested. Maybe at the end of the intro, direct readers to the tutorials on strands and specify that the rest of this document is for contribution?